internal/kafka/test/integration/kafkas_test.go

```diff
@@ -724,7 +728,8 @@ func testValidations(standardClusters, developerClusters string, h *coreTest.Hel
 				}
 			}
 		} else {
-			checkMetricsError := common.WaitForMetricToBePresent(h, t, metrics.ClusterStatusCapacityUsed, val.capacityUsed, val.kafka.InstanceType)
+			Expect(val.kafka.ClusterID).To(Equal(val.assignedClusterID))
+			checkMetricsError := common.WaitForMetricToBePresent(h, t, metrics.ClusterStatusCapacityUsed, val.capacityUsed, val.kafka.InstanceType, val.assignedClusterID, val.kafka.Region, val.kafka.CloudProvider)
 			Expect(checkMetricsError).NotTo(HaveOccurred())
```

Comment:
```suggestion
			Expect(common.WaitForMetricToBePresent(h, t, metrics.ClusterStatusCapacityUsed, val.capacityUsed, val.kafka.InstanceType, val.assignedClusterID, val.kafka.Region, val.kafka.CloudProvider)).
			To(Succeed())
```

Reply:
Thanks. I'll keep the earlier version as the suggested version seem longer to me given the variable taken by the metrics polling.

Reply:
discussed via chat